pkg/apis/scorecard/v1alpha2/formatter.go

```diff
@@ -46,27 +46,37 @@ func (s ScorecardOutput) MarshalText() (string, error) {
 			sb.WriteString(fmt.Sprintf("%s:\n", result.Labels["suite"]))
 			currentSuite = result.Labels["suite"]
 		}
-		sb.WriteString(fmt.Sprintf("\t%-35s: ", result.Name))
+		sb.WriteString(fmt.Sprintf("\t%-35s ", result.Name))
```

Comment:
A couple of major nits coming at you... 🙂 

1. Should we do an initial loop through the results to figure out the longest `result.Name` and use that for the padding?

    I think it would be `fmt.Sprintf("\t%-*s", maxWidth, result.Name)`

2. Should the padding only be applied at the time we print the state? This could be applicable in the list scenario if we print some extra blank characters on a line that cause the terminal to wrap to the next line with a blank character or two, which might happen when the terminal width the right at or just above the `maxWidth`. So maybe we'd end up doing:
    ```
    fmt.Sprintf("%*s", maxWidth-len(result.Name), fmt.Sprintf(passColor, PassState))
    ```

Reply:
same as above, if its ok, lets push this formatting to the story work on formatting.

Reply:
Yep. That's fine!